server/views/manage/premises/placements/show.njk

```diff
@@ -41,27 +44,57 @@
         },
         menus: PlacementUtils.actions(placement, user)
     }) }}
+    {{ mojSubNavigation({
+        label: 'Sub navigation',
+        items: tabItems
+    }) }}
+    {% if activeTab=='timeline' %}
+        <h2 class="govuk-heading-m">Placement timeline</h2>
+        {{applicationTimeline(timelineEvents)}}
+    {% elseif activeTab==='application' %}
+        {{ applicationReadonlyView(application) }}
+    {% elseif activeTab==='assessment' %}
+      {% for section in SummaryListUtils.summaryListSections(assessment, false) %}
+        <h2 class="govuk-heading-l">{{ section.title }}</h3>
+        {% for task in section.tasks %}
+          {{
+            govukSummaryList(task)
+          }}
+        {% endfor %}
+      {% endfor %}
```

Comment:
There's a mismatched `<h2>` and `</h3>` closing tag in there.

Feels like there should be a macro for this!

Reply:
Also, why didn't the accessibility checker spot this?

Reply:
That is a good question 🤔